### DISCUSSION TITLE
Yellow tags

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1051] [General]
Hi
How to select those yellow tags (CYCLO ROAD) only?
Thank you
![2023-04-02 11_36_12-Livre des parcours - MyTourbook - 23 3 0](https://user-images.githubusercontent.com/4082252/229344939-b53548a6-1819-45d9-b3c2-4b09752fc21a.jpg)

**Comment by Thompson-ongithub** [comment]
If I'm not mistaken, that's the Tour Type, what do you want to select?

**Comment by FJBDev** [comment]
> How to select those yellow tags (CYCLO ROAD) only?

You will need to select the "CYCLO ROAD" tour type in tour type filter just as explained in the MT documentation [here](https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/tools/tour-type#select-tour-type)

**Comment by gringojack** [comment]
It is so simple, thank you guys!
